src/test/java/com/wikia/webdriver/testcases/mercurytests/CategoryTest.java

```diff
@@ -16,19 +14,12 @@
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
 public class CategoryTest extends NewTestTemplate {
 
-  private Category category;
-
-  private void init() {
-    this.category = new Category(driver);
-
-    new Navigate(driver).toPage(MercurySubpages.MAIN_PAGE);
-  }
-
-  @Test(groups = "mercury_category_expandAndNavigateToCategoryPage")
-  public void mercury_category_expandAndNavigateToCategoryPage() {
-    init();
-
-    category.toggle();
-    category.openCategoryPage(0);
+  @Test(groups = "mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage")
+  public void mercury_category_navigateToCategoryPageFromCategoryComponentOnMainPage() {
+    new MainPage()
+        .openRegularMainPage()
+        .useCategoryComponent()
```

Comment:
rename it from useCategoryComponent to getCategoryComponent - without going to code of this method I had no idea what it does. Or even better don't chain if you change the type